### PR TITLE
Prettyify dates in 21-0966

### PR DESCRIPTION
**Pull request by ndsprinkle**
## Summary
This pull request makes all the expiration dates and submission dates in 21-0966 "pretty" by formatting them in a readable format.

## Related issue(s)
department-of-veterans-affairs/va.gov-team-forms#999
department-of-veterans-affairs/va.gov-team-forms#1068

## Screenshots
image

### CODE REVIEW

File: src/applications/simple-forms/21-0966/containers/ConfirmationPage.jsx [hunk 1]
```diff
@@ -121,7 +125,12 @@ export const ConfirmationPage = props => {
                 <p>
                   Our records show that you already have an intent to file for
                   pension claims. Your intent to file for pension claims expires
-                  on {data['view:activePensionITF'].expirationDate}.
+                  on{' '}
+                  {format(
```
Comment (rhasselle-oddball) [inline src/applications/simple-forms/21-0966/containers/ConfirmationPage.jsx:129]: what's the difference between the dates that have `format` and the ones that don't?

**Review by rhasselle-oddball — APPROVED** [review]
looks good to me - one question -

File: src/applications/simple-forms/21-0966/containers/ConfirmationPage.jsx (hunk 1 above)
Comment (ndsprinkle) [reply src/applications/simple-forms/21-0966/containers/ConfirmationPage.jsx:129]: Theoretically they should all be formatted. Are you seeing one that isn't?

Comment (ndsprinkle) [reply src/applications/simple-forms/21-0966/containers/ConfirmationPage.jsx:129]: The two individual benefit Yes/No pages have formatted expiration dates and the confirmation page (and its helpers) should have a few too.

**Pull request merged**